server/server.go

```diff
@@ -0,0 +1,65 @@
+package server
+
+import (
+	"net/http"
+	"github.com/gruntwork-io/health-checker/options"
+	"net"
+	"fmt"
+)
+
+func StartHttpServer(opts *options.Options) {
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Should the path be configurable? E.g., `/health`? Another option that may be useful in the future for debugging is `/health/<port>` to check just a single port.